sdk/communication/communication-rooms/test/public/roomsClient.spec.ts

```diff
@@ -128,6 +128,23 @@ describe("RoomsClient", function () {
       await client.updateRoom(roomId, options);
     });
 
+    it("successfully list rooms", async function () {
+      const listRoomsResult = await client.listRooms({});
+
+      assert.isDefined(listRoomsResult);
+      assert.isNotEmpty(listRoomsResult);
+      for await (const roomModel of listRoomsResult) {
+        if (roomModel) {
+          assert.isDefined(roomModel);
```

Comment:
no need for this assertion since you used `roomModel` as the if condition

Reply:
right, I was trying to understand the behavior of PagableAsync, I will just simply call assert.isDefined(roomModel)